test/built-ins/object/values/tamper-with-global-object.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2015 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+    Object.values should not have its behavior impacted by modifications to the global property Object
+es7id: pending
+author: Jordan Harband
+---*/
+
+function fakeObject() {
+    $ERROR('The overriden version of Object was called!');
+}
+
+var global = Function('return this;')();
```

Comment:
Preferable: Include fnGlobalObject.js and use `fnGlobalObject()` to retrieve the global object.